Makefile

```diff
@@ -1,5 +1,7 @@
 all:
 	dune build
+depend:
```

Comment:
We needed depend when we were using Makefiles to build pfff, but now that we use dune,
the dependencies are managed automatically so you can remove this target
(installing the dependencies of the project is another thing; it should not be confused with what
most OCaml programmers expect from a 'make depend').